xml/cha_administration.xml

```diff
@@ -28,4 +28,187 @@
   </dm:docmanager>
  </info>
 
+ <sect1 xml:id="sec-admin-register">
+  <title>Registering instances</title>
+  <para>
+   Like other &suse; products, &sle; in public clouds has to be registered to
+   receive updates. There are different ways to register, depending on what
+   instance you register where.
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     <literal>PAYG</literal> instances are registered against a local update
+     server, that means an update server in the cloud framework and region. This
+     is done automatically by the <systemitem
+      class="service">guestregister.service</systemitem> upon first boot.
+    </para>
+  </listitem>
+   <listitem>
+    <para>
+     <literal>BYOS</literal> instances have to be registered manually with your
+     &suse; registration key. You can register against a local update server,
+     the &scc;, or your own &susemgr; or &rmt; infrastructure.
+    </para>
+   </listitem>
+  </itemizedlist>
+  <para>
+   There are two different ways to register BYOS instances with the &scc;,
+   depending on the datestamp of the image the instance was created from:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Images with a datestamp <emphasis>later</emphasis> than
+     <literal>20220103</literal> can be registered with
+     <command>registercloudguest</command>. This sets up
+     <command>&suseconnect;</command> to work against the cloud framework's update
+     infrastructure rather than the &scc;, resulting in significantly faster
+     package downloads. Refer to
+     <xref linkend="sec-admin-registercloudguest"/> for instructions.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Images with a datestamp <emphasis>prior</emphasis> to
+     <literal>20220103</literal> are registered with
+     <package>&suseconnect;</package>. Refer to <xref
+      linkend="sec-admin-suseconnect"/>. Alternatively, you can install
+     <command>registercloudguest</command> to benefit from faster downloads
+     from the local cloud update infrastructure.
+    </para>
+   </listitem>
+  </itemizedlist>
+  <para>
+   In summary, use <command>registercloudguest</command> to register with the
+   &scc;; it provides significant benefits even for older instances.
+   Use <command>&suseconnect;</command> to register with or your own &susemgr;
+   or &rmt; infrastructure.
```

Comment:
It can, but this summary is our recommendation. If there is no reason to use SUSEConnect and SCC, we should recommend registercloudguest and the cloud update infrastructure for faster downloads. Right?

Reply:
Added in 3bcfc5f165255f7e860343998dc073de8f98c024.